### DISCUSSION TITLE
有人做过动态壁纸吗？有没有手把手的视频？我想做一个

### DISCUSSION THREAD

**Discussion by youcall911** [t1m0thyj/WinDynamicDesktop#603] [General]
有人做过动态壁纸吗？有没有手把手的视频教程？我想做一个

**Comment by t1m0thyj** [comment]
> Has anyone made a dynamic wallpaper? Is there a step-by-step video tutorial? I want to make one

@youcall911 Please see the steps here for creating your own theme: [Creating custom themes](https://github.com/t1m0thyj/WinDynamicDesktop/wiki/Creating-custom-themes)

Someone has made a video that you may also find helpful: https://www.youtube.com/watch?v=_pnF66tctQo
